docs/src/main/sphinx/ext/redirects.py

```diff
@@ -0,0 +1,86 @@
+#
+#   sphinxcontrib.redirects
+#   ~~~~~~~~~~~~~~~~~~~~~~~
+#   Generate redirects for moved pages when using the HTML builder.
+#   :copyright: Copyright 2017 by Stephen Finucane <stephen@that.guru>
+#   :license: BSD, see LICENSE for details.
+#   https://github.com/sphinx-contrib/redirects/blob/master/LICENSE
+#
+#   Modified April 10, 2023 for Trino use
```

Comment:
How? Maybe link to issue and also detail what version was modified .. so we can figure it out a bit easier in the future